sklearn/svm/classes.py

```diff
@@ -1061,12 +1069,27 @@ def decision_function(self, X):
 
         Returns
         -------
-        X : array-like, shape (n_samples,)
+        dec : array-like, shape (n_samples,)
             Returns the decision function of the samples.
         """
         dec = self._decision_function(X)
         return dec
 
+    def score_samples(self, X):
+        """Raw decision function of the samples.
```

Comment:
Im not very pleased with `Raw decision function`. Maybe `Shifted decision function` or `scoring function`?